request-server.go

```diff
@@ -288,9 +288,9 @@ func cleanPath(p string) string {
 }
 
 func cleanPathWithBase(base, p string) string {
-	p = filepath.ToSlash(p)
+	p = filepath.ToSlash(filepath.Clean(p))
```

Comment:
The order here really doesn’t matter except for Windows, and only for UNC paths. “Let’s just not support UNC paths“ doesn’t work, because OpenSSH’s client does not sanitize the path first, so UNC paths would still work with that client anyways. :sob:

Reply:
```
linux-machine $ sftp -P 31337 windows-machine
sftp> ls //wsl$/openSUSE-Leap-15.3
//wsl$/openSUSE-Leap-15.3/bin           //wsl$/openSUSE-Leap-15.3/dev           //wsl$/openSUSE-Leap-15.3/etc           
//wsl$/openSUSE-Leap-15.3/home          //wsl$/openSUSE-Leap-15.3/init          //wsl$/openSUSE-Leap-15.3/lib           
//wsl$/openSUSE-Leap-15.3/lib64         //wsl$/openSUSE-Leap-15.3/lost+found    //wsl$/openSUSE-Leap-15.3/mnt           
//wsl$/openSUSE-Leap-15.3/opt           //wsl$/openSUSE-Leap-15.3/proc          //wsl$/openSUSE-Leap-15.3/root          
//wsl$/openSUSE-Leap-15.3/run           //wsl$/openSUSE-Leap-15.3/sbin          //wsl$/openSUSE-Leap-15.3/selinux       
//wsl$/openSUSE-Leap-15.3/srv           //wsl$/openSUSE-Leap-15.3/sys           //wsl$/openSUSE-Leap-15.3/tmp           
//wsl$/openSUSE-Leap-15.3/usr           //wsl$/openSUSE-Leap-15.3/var
```
:woman_shrugging: alright.